production/loki-mixin/dashboards/loki-chunks.libsonnet

```diff
@@ -82,24 +63,108 @@ local utils = import 'mixin-utils/utils.libsonnet';
       $.row('Flush Stats')
       .addPanel(
         $.panel('Queue Length') +
-        $.queryPanel('cortex_ingester_flush_queue_length{%s}' % dashboards['loki-chunks.json'].ingesterSelector, '{{pod}}'),
+        $.queryPanel('cortex_ingester_flush_queue_length{%s}' % labelsSelector, '{{pod}}'),
       )
       .addPanel(
         $.panel('Flush Rate') +
-        $.qpsPanel('loki_ingester_chunk_age_seconds_count{%s}' % dashboards['loki-chunks.json'].ingesterSelector,),
+        $.qpsPanel('loki_ingester_chunk_age_seconds_count{%s}' % labelsSelector,),
       ),
     )
+    .addRow(
+      $.row('Flush Stats')
+      .addPanel(
+        $.panel('Chunks Flushed/Second') +
+        $.queryPanel('sum(rate(loki_ingester_chunks_flushed_total{%s}[$__rate_interval]))' % labelsSelector, '{{pod}}'),
```

Comment:
note that this query gets us the same result as the one immediately above on line 70, but this metric seems more accurate (?), correct(?)